app/src/main/java/com/ivy/wallet/IvyAndroidApp.kt

```diff
@@ -31,4 +36,26 @@ class IvyAndroidApp : Application(), Configuration.Provider {
             Timber.plant(DebugTree())
         }
     }
-}
\ No newline at end of file
+}
+
+// This solves Could not instantiat worker issue
+class CustomWorkerFactory @Inject constructor(
```

Comment:
Is there a better solution to this? This seems like a hack that'll become tech debt in the future